src/ServiceManagerBase.sol

```diff
@@ -135,6 +137,114 @@ abstract contract ServiceManagerBase is ServiceManagerBaseStorage {
         _setRewardsInitiator(newRewardsInitiator);
     }
 
+    function migrateOperatorSetIds(uint32[] memory operatorSetsToCreate) external onlyOwner{
+        _migrateCreateOperatorSetIds(operatorSetsToCreate);
+    }
+
+    function migrateToOperatorSets(uint32[][] memory operatorSetIds, address[] memory operators) external onlyOwner {
+        require(!migrationFinalized, "SerivceManager: Migration Already Finalized");
```

Comment:
Is this require statement needed? Seems like it's not giving much of a guarantee?

Reply:
Or, do we want to just to be explicit when avs has migrated ig?

Reply:
Before it wasn't possible to do two transactions because `becomeOperatorSetAVS` blocked you from doing the second call and effectively resulted in a finalization guarantee of the migration.  But now that we needed to do 2 calls, the becomeOperatorSetAVS check happens separately in the `migrateOperatorSetIds` function above